ding/framework/middleware/functional/logger.py

```diff
@@ -3,7 +3,7 @@
 from ding.utils import DistributedWriter
```

Comment:
add unittest for this file, and you can ask for @HansBug about methods to test logger output in pytest